05/components/ContactForm.js

```diff
@@ -0,0 +1,139 @@
+/* eslint-disable */
+import React, { useReducer } from 'react';
+import { v4 as uuid } from 'uuid';
+import FormItem from './FormItem';
+import ValidationRules from '../FormHelpers/ValidationRules';
+import FormValidator from '../FormHelpers/FormValidator';
+import account from '../account';
+// import propTypes from 'prop-types';
+
+const ContactForm = (props) => {
+    const initState = {
+        firstName: '',
+        lastName: '',
+        email: '',
+        phone: '',
+        title: '',
+        message: '',
+        errors: [],
+    };
+    const { formFields } = props;
+    const formFieldsNames = formFields.map(field => field.name)
+
+    function reducer(state, action) {
+        switch (action.type) {
+        case action.type: {
+            return {
+                ...state,
+                [action.type]: action.payload,
+            };
+        }
```

Comment:
Oj, oj oj... takich brzydkich rzeczy nie robimy :P
Po co w takim razie jakiś warunek jeśli on zawsze jest spełniony... to jak sprawdzać czy `` === 1` :P
Dlatego czyszczenie formularza nie działa ;)